### PR TITLE
Custom search box text [PD-712]

### PR DESCRIPTION
**Pull request by almathew**
All tests pass

### CODE REVIEW

File: templates/search_box.html [hunk 1]
```diff
@@ -8,18 +8,16 @@
...
-
-        </div>
-        <div class="searchTab">
-          <label for="q">{% trans 'What' %}</label>
-          <input title="Search Phrase" type="text" name="q" id="q" class="micrositeTitleField" value={% if title_term != "\*" %}"{{ title_term }}"{% else %}""{% endif %} />
-          <span class="helptext" id="what">{% blocktrans %}job title, keywords{% endblocktrans %}</span>
-        </div>
-        
+          <div class="searchTab">
+              <label for="location">{% if site_config.where_label %}{{ site_config.where_label }}{% else %}{% trans "Where" %}{% endif %}</label>
```
Comment (aspidites) [inline templates/search_box.html]: Will a default filter not work? Something like `{{ site_config.where_label|default:"Where" }}`

Comment (aspidites) [inline templates/search_box.html]: Oh. Trans. Ugh.

Comment (almathew) [inline templates/search_box.html]: Yeah. I think they actually use it too. I'm afraid to find out.

Comment (aspidites) [inline templates/search_box.html]: http://stackoverflow.com/questions/3595099/django-trans-tag-within-a-default-filter

**Comment by almathew** [comment]
I've addressed @aspidites's comments a bit, but the place that probably needs it the most (help text) will need to be a multi-line change in order to continue marking it as safe. Opinions?

**Comment by aspidites** [comment]
@almathew : Out of curiosity, is this where bleach would come in? That is, if we are concerned about marking things as safe, would bleach resolve those concerns, or be over kill? (Yay, super slow rendering because of bleach!).

**Comment by almathew** [comment]
I'm marking it as safe to contain html.

**Comment by aspidites** [comment]
I thought bleach did that, but removed unsafe html. That whole cake and eating it too, thing?

**Pull request merged**